sumologic/resource_sumologic_muting_schedules_library_muting_schedule.go

```diff
@@ -106,20 +109,26 @@ func getScheduleDefinitionSchemma() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			ForceNew: false,
 			Required: true,
-			ValidateFunc: validation.StringMatch(regexp.MustCompile(`^\d{4}-\d{2}-\d{2}$`),
-				"start date in format of yyyy-mm-dd"),
+			ValidateFunc: validation.All(
+				validation.StringMatch(regexp.MustCompile(`^\d{4}-\d{2}-\d{2}$`),
+					"start date in format of yyyy-mm-dd"),
+				StringIsValidStartDate(),
```

Comment:
Curious why this is needed?

Reply:
The start date should be greater than the current date - 24 hours(buffer time). We do not allow the user to create a muting scheduler that starts past.